.github/workflows/ci.yml

```diff
@@ -39,11 +39,11 @@ jobs:
           docker compose up -d
           sleep 10
       - name: Create Test Database
-        run: docker compose -p redash run --rm postgres psql -h postgres -U postgres -c "create database tests;"
+        run: docker compose run --rm postgres psql -h postgres -U postgres -c "create database tests;"
```

Comment:
I'm not very knowledgeable about this part, so please let me know if I'm wrong.

Currently, Redash CI runs tests in multiple pipelines (backend / frontend), and it is necessary to clearly separate container instances in each.

`-p` is appended to the container name and must be clearly distinguished for each task to be executed. Therefore, I don't think you should remove this.

Reply:
-p options is the same as COMPOSE_PROJECT_NAME env variable, which is set globally for a whole CI job

Reply:
Thanks for separating the PR. Please discussing this in https://github.com/getredash/redash/pull/6674.